contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.2.0--4.3.0.sql

```diff
@@ -37,6 +37,10 @@ LANGUAGE plpgsql;
  * So make sure that any SQL statement (DDL/DML) being added here can be executed multiple times without affecting
  * final behaviour.
  */
+SET allow_system_table_mods = ON;
+ALTER TABLE sys.babelfish_function_ext DROP CONSTRAINT babelfish_function_ext_pkey;
+ALTER TABLE sys.babelfish_function_ext ADD PRIMARY KEY (funcname, nspname, funcsignature);
```

Comment:
Let's use name constraint just to be on safer side

Reply:
Primary key constraint name is always fixed.

Reply:
This works:
`alter table mytable add constraint my_pk primary key (mycol)`